src/app/editor/code/editor-container/editor-container.component.ts

```diff
@@ -22,7 +22,7 @@ export class EditorContainerComponent implements OnInit, AfterContentInit {
     logo: FunctionStringCallback;
   }>;
 
-  constructor(private codegen: CodeGenService, private readonly store: Store) {}
+  constructor(private codegen: CodeGenService, private store: Store) {}
```

Comment:
why did we remove the `readonly` keyword? This prevents us from accidentally mutating the `store` ref.

Reply:
I didn't know, at some point I started readonly everywhere for no reason.